src/plugin.js

```diff
@@ -364,8 +364,7 @@ const vr = function(options) {
       camera.updateProjectionMatrix();
     }
 
-    window.addEventListener('resize', onWindowResize, false);
-    window.addEventListener('vrdisplaypresentchange', onWindowResize, true);
+    player.on(['resize', 'fullscreenchange'], onWindowResize);
```

Comment:
I guess there isn't any harm in listening to all of these events (minus resize on player)